mos-platform/common/lib/exit.c

```diff
@@ -2,7 +2,9 @@
 #include <stdio.h>
 
 extern __attribute__((noreturn)) void __exit(int status);
+extern void __fini();
 
 void exit(int status) {
+  __fini();
```

Comment:
`abort()` currently calls through to `exit`, so as written, this will cause `abort()` to call functions registered with `atexit()`, in violation of the standard.

Reply:
Unless I'm missing something, this is a bug isn't it?  Abort() shouldn't call through exit(); it should terminate immediately.

Reply:
It's a bug with this PR, but it wasn't before. Previously, exit had no behavior besides terminating.

Reply:
OK; so I should modify abort() so it goes straight to __exit()?

Reply:
Yeah, SGTM